src/posh-git.psd1

```diff
@@ -1,10 +1,10 @@
 @{
 
 # Script module or binary module file associated with this manifest.
-ModuleToProcess = 'posh-git.psm1'
+RootModule = 'posh-git.psm1'
 
 # Version number of this module.
-ModuleVersion = '0.7.0'
+ModuleVersion = '1.0.0'
```

Comment:
Yes, but doing that if posh-git has already been loaded, e.g. in the GitHub Desktop shell, leaves you with two posh-git modules loaded. If one of those is 0.7.0 and the other is a personal fork that also reports 0.7.0, I have to check path to differentiate. Thus my suggestion of the extra `.0` as a hint that this isn't an official version.

That said, also including `PrereleaseVersion` makes sense… add that to the issue template script?